docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_general.scss

```diff
@@ -69,6 +69,16 @@ h6 {
   margin-top: 40px;
   font-style: normal;
 }
+span.event-series__header {
+  @extend h4;
+  display:block;
+}
+span.event-title__header {
+  @extend h3;
+  display:block;
+  line-height: 1.25em;
+  font-size: 1.25em;
+}
```

Comment:
can you move this to https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/entrypoints/event/event-legacy.scss 

This is a much better place since it is for events. 

One additional note for others, there was never an `.event-title` class defined in any of our css files ever. or else I would have asked for that class to to be replaced... It would have gone into events-legacy.scss most likely.

Reply:
updated to new _event.scss